core/src/main/resources/defaults/clustertemplates/7.2.9/aws/dataengineering-ha.json

```diff
@@ -11,6 +11,7 @@
     "externalDatabase": {
       "availabilityType": "HA"
     },
+    "enableLoadBalancer": true,
```

Comment:
Along with this, does the manager need to be changed to type CORE, and the master to GATEWAY. That is at least what 7.2.8 does. Unless that change is expected in the change @smaniraju is currently making.

Reply:
Yeah, I am relying on @smaniraju's change.

Reply:
Yes, particularly I did not want to introduce merge conflicts for @smaniraju